src/components/Message.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import { StyleSheet, View, Text, ViewStyle, StyleProp } from 'react-native'
+
+import LargeButton, { LargeButtonProps } from '../components/LargeButton'
+import textStyle from '../styles/text'
+import theme from '../styles/theme'
+
+type Props = {
+  title?: string
+  text: string
+  buttons?: LargeButtonProps[]
+  style?: StyleProp<ViewStyle>
+}
+
+const Message = ({ title, text, buttons, style }: Props) => (
+  <View style={[styles.message, style]}>
+    {title && (
+      <View style={{ marginBottom: theme.margin.half }}>
+        <Text style={textStyle.lead}>{title}</Text>
+      </View>
+    )}
+    <View>
+      <Text style={textStyle.body}>{text}</Text>
+    </View>
+    {buttons && buttons.length > 0 && (
+      <View
+        style={{
+          marginTop: theme.margin.common,
+          flexDirection: 'row',
+          alignItems: 'flex-start',
+        }}
+      >
+        {buttons.map((button, i) => (
+          <LargeButton
+            disabled={button!.disabled || false}
+            secondary={button!.secondary || false}
+            title={button!.title}
+            key={i}
```

Comment:
De `key` was eerst de title, maar in de unit test "Message - Rendering - Without title" kregen we deze foutmelding: 
`Warning: Each child in a list should have a unique key prop`.

Reply:
Normaal gesproken bad practice. De key moet corresponderen met het item zelf, niet met de plek van het item in de lijst. Maar als je identieke `LargeButtonProps[]` kan doorgeven, dan heb je geen info om een unieke key te maken.

Reply:
In https://www.reactnative.express/react/lists_and_keys:
_This silences the warning from React about forgetting to include keys, but remember that doing this will cause React to identify elements incorrectly if the data is modified: for example, if a new item is inserted at the front of the list, it will get key '0', which previously belonged to another item. So you may be better off assigning identifiers or indexes to your data set if you can._

Good to know!